tsconfig.json

```diff
@@ -68,5 +68,6 @@
     /* Advanced Options */
     "skipLibCheck": true,                           /* Skip type checking of declaration files. */
     "forceConsistentCasingInFileNames": true        /* Disallow inconsistently-cased references to the same file. */
-  }
+  },
+  "include": ["src"]
```

Comment:
Needed so TS picks up [the dom-types](https://github.com/escapedcat/lightning-browser-extension/blob/chore/584_testing-library-jest-setup/src/setupTests.ts#L5)